ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/router_ospf.schema.yml

```diff
@@ -298,7 +298,5 @@ keys:
             mpls_ldp_sync_default:
               type: bool
             eos_cli:
-              type: list
-              items:
-                type: str
-                description: The eos cli command
+              type: str
+              description: Multiline EOS CLI rendered directly on the router ospf in the final EOS configuration
```

Comment:
It's under a process id which is under router ospf. So I would just clarify in comment `Multiline EOS CLI rendered directly on the router ospf process id in the final EOS configuration`